### PR TITLE
Added VSL chain info

### PR DESCRIPTION
**Pull request by alexander-shustanov**
The blockchain is used by the Vessel network to maintain an active registry of digital goods in mobile apps.

### CODE REVIEW

**Comment by easmith** [comment]
👍

**Comment by alexander-shustanov** [comment]
The blockchain network is used by the Vessel Wallet app to account for in-game items.

File: _data/icons/vsl.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url":"ipfs://QmeknNzGCZXQK7egwfwyxQan7Lw8bLnqYsyoEgEbDNCzJX",
```
Comment (ligi) [inline _data/icons/vsl.json:3]: where did you pin this CID? I cannot resolve it

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/vsl.json (hunk 1 above)
Comment (alexander-shustanov) [reply _data/icons/vsl.json:3]: I pinned it with pinata
You can check it here https://gateway.pinata.cloud/ipfs/QmeknNzGCZXQK7egwfwyxQan7Lw8bLnqYsyoEgEbDNCzJX

File: _data/chains/eip155-7355310.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "symbol": "VETH",
+    "decimals": 18
+  },
+  "infoURL": "https://www.openvessel.io/",
+  "shortName": "vsl",
+  "chainId": 7355310,
+  "networkId": 7355310,
+  "explorers": [{
+    "name": "openvessel-mainnet",
+    "url": "https://mainnet-explorer.openvessel.io/",
```
Comment (ligi) [inline _data/chains/eip155-7355310.json:20]:
```suggestion
    "url": "https://mainnet-explorer.openvessel.io",
```

**Comment by ligi** [comment]
I could resolve it now - but somehow there seems to be a problem with IPFS - before I tried to resolve it for a very long time with the client - and it did not resolve.

please run the prettier and then I can merge

**Comment by alexander-shustanov** [comment]
> I could resolve it now - but somehow there seems to be a problem with IPFS - before I tried to resolve it for a very long time with the client - and it did not resolve.
> 
> please run the prettier and then I can merge

done

**Comment by alexander-shustanov** [comment]
@ligi Hi! Looks like all changes already made. Can you merge it?

**Comment by alexander-shustanov** [comment]
@ligi Is there anything I should do to merge PR?

**Comment by ligi** [comment]
prettier check still does not seem to pass ..

**Review by ligi — APPROVED** [review]

**Pull request merged**